integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/K8sEvents.java

```diff
@@ -174,7 +175,10 @@ public static boolean domainEventExists(
     try {
       List<CoreV1Event> events = Kubernetes.listNamespacedEvents(domainNamespace);
       for (CoreV1Event event : events) {
-        if (event.getReason().equals(reason) && (isEqualOrAfter(timestamp, event))) {
+        if (((domainUid != null && (event.getMetadata().getLabels() != null
```

Comment:
possible NullPointerException here. You need to check event != null && event.getMetadata() != null too